packages/core/src/helper/eventGroups.ts

```diff
@@ -0,0 +1,60 @@
+import { RoomEvent } from 'livekit-client';
+
+export type UpdateOnlyOn = (RoomEvent[] | RoomEvent)[];
+
+export function toRoomEvents(updateOnlyOn: UpdateOnlyOn): RoomEvent[] {
+  const events: RoomEvent[] = [];
+  updateOnlyOn.forEach((value) => {
+    if (Array.isArray(value)) {
+      events.concat(value);
+    } else {
+      events.push(value);
+    }
+  });
+  return events;
+}
+
+/**
+ * RoomEventGroup is a helper object of RoomEvents that are logically grouped together.
+ * The intention is to provide a high-level way to get all the RoomEvents needed to subscribe to certain state changes.
+ *
+ * @example
+ * ```tsx
+ *   // Get all events needed to listen to tack updates.
+ *   const events = RoomEventGroup.trackUpdates
+ * ```
+ */
+export const RoomEventGroup: Record<string, RoomEvent[]> = {
```

Comment:
I think we'll need to do a pass on naming. 
both for the "group" and also for its members. 

I'd be in favor of naming it around the Participants from the start, as - at leaast in this case - the use case is on the useParticipants hook.

Reply:
I'm not a fan of the names either. I'm happy to hear your suggestions.